KarhooUISDK/Screens/CheckoutScreen/Checkout/DetailsCellView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  DetailsViewCell.swift
+//  KarhooUISDK
+//
+//  Created by Bartlomiej Sopala on 03/11/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct DetailsCellView: View {
+    
+    @ObservedObject var model: DetailsCellModel
+    weak var delegate: DetailsCellViewControllerProtocol?
+
+    var body: some View {
+        Group {
+            HStack(spacing: UIConstants.Spacing.small) {
+                Image(uiImage: UIImage.uisdkImage(model.iconName))
+                .frame(
+                    width: UIConstants.Dimension.Icon.large,
+                    height: UIConstants.Dimension.Icon.large
+                )
+                .foregroundColor(KarhooUI.colors.primary.getColor())
+            VStack(alignment: .leading) {
+                Text(model.title)
+                    .lineLimit(2)
+                    .font(.system(size: 16))
+                    .foregroundColor(KarhooUI.colors.text.getColor())
+                Text(model.subtitle)
+                    .lineLimit(2)
+                    .font(.system(size: 10))
```

Comment:
let's use .font(.caption2) or the closest option available

Reply:
Changed to: `.font(Font(KarhooUI.fonts.footnoteRegular()))`